seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
             case BOOLEAN:
-                return Bytes.toBoolean(cell);
+                return Boolean.valueOf(Bytes.toString(cell));
             case BIGINT:
-                return Bytes.toLong(cell);
+                return Long.valueOf(Bytes.toString(cell));
```

Comment:
**Issue**: Converting bytes to string and then to long is inefficient and error-prone if the string is not a valid long. <br> **Fix**: Revert to direct byte to long conversion to avoid performance overhead and potential conversion errors. <br> **Code Suggestion**: 
 ```
 -                return Long.valueOf(Bytes.toString(cell));
 +                return Bytes.toLong(cell);
 ```